gammapy/spectrum/sensitivity.py

```diff
@@ -35,7 +39,6 @@ class SensitivityEstimator(object):
     --------
     Compute and plot a sensitivity curve for CTA::
 
-        from gammapy.irf import CTAPerf
```

Comment:
You removed the import statement here, but the rest of the example still uses `CTAPerf`. Either fix the example or remove it completely and link to the notebook instead? Both options are fine by me...

Reply:
I removed the example. You can't show how this works in only 4 lines of code now.